src/devices/src/virtio/mmio.rs

```diff
@@ -316,9 +318,10 @@ impl BusDevice for MmioTransport {
             }
             _ => {
                 warn!(
-                    "invalid virtio mmio write: 0x{:x}:0x{:x}",
+                    "invalid virtio mmio write: 0x{:x}:0x{:x} for device type [{}]",
                     offset,
-                    data.len()
+                    data.len(),
+                    self.device().lock().expect("Poisoned Lock!").device_type()
```

Comment:
During development I was getting this warning. Seeing it is for a balloon device (which shouldn't have been used in the first place) and not for a virtio-mem device helped me narrow down the search for what I was doing wrong.

This was the best information I could get at that point, as getting the device id (which could identify between those 5 hypothetical block devices) is not possible there.

With this in mind I think it would be helpful to keep it for a while.